ludwig/models/llm.py

```diff
@@ -193,6 +193,9 @@ def initialize_adapter(self):
             self.model.print_trainable_parameters()
             logger.info("==================================================")
 
+    def prepare_for_training(self):
+        self.initialize_adapter()
```

Comment:
Would add a TODO comment that this will not work if resuming from previous checkpoint.

Reply:
Done.